src/components/results/sensitivity-analysis/sensitivity-analysis-result.js

```diff
@@ -119,6 +158,8 @@ const SensitivityAnalysisResult = ({
         () => ({
             suppressMovable: true,
             sortable: true,
+            resizable: true,
+            flex: 1,
```

Comment:
why flex: 1 ? when  we have   const onGridReady = useCallback((params) => {
        if (params.api) {
            params.api.sizeColumnsToFit();
        }
    }, []);
    
    do you think that it is necessary?

Reply:
Done